example/module/sequential_module.py

```diff
@@ -0,0 +1,60 @@
+# pylint: skip-file
+import mxnet as mx
+import numpy as np
+import logging
+
+# whether to demo model-parallelism + data parallelism
+demo_data_model_parallelism = True
+
+if demo_data_model_parallelism:
+    contexts = [[mx.context.gpu(0), mx.context.gpu(1)], [mx.context.gpu(2), mx.context.gpu(3)]]
+else:
+    contexts = [mx.context.cpu(), mx.context.cpu()]
+
```

Comment:
nice, thanks. i'm looking if we can get a machine translation example (embedding+lstm+attention) and test the model parallelism performance.